jsonnet/thanos-mixin/alerts/absent.libsonnet

```diff
@@ -0,0 +1,25 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'component-absent',
```

Comment:
This is not very specific. We should at least prefix with `thanos-` or something.